### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by devrungta**
- Added the UIs for log screen, proctor's home screen, student's home screen, signup page but some logic is left.
-  Made the models for logs, students.
- 
flutter_02
flutter_01
flutter_03

### CODE REVIEW

File: client/pubspec.yaml [hunk 1]
```diff
@@ -59,8 +64,9 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+  assets:
+  -  C:\Users\devru\Digital-Entry-System\client\lib\Assets\iit_logo_original.png
+  #Yaar relative path kaam hi nhi kar rha tha mere laptop me isliye maine absolute path daal diya.
```
Comment (criticic) [inline client/pubspec.yaml:69]: `- lib/Assets/iit_logo_original.png`
does this not work? it seems to work for me

Comment (devrungta) [reply client/pubspec.yaml:69]: Mene to try Kara tha, mereme kaam nhi kra, doesn't exist aa rha tha😭😭

Comment (Rasesh2005) [reply client/pubspec.yaml:69]: > `- lib/Assets/iit_logo_original.png` does this not work? it seems to work for me

this works for me too.

File: client/lib/Screens/signup.dart
Comment (Rasesh2005) [inline client/lib/Screens/signup.dart:1]: Assertion failed: file:///Users/rasesh/Developer/flutter/packages/flutter/lib/src/widgets/image.dart:479:10
IklsWeb
"Image.file is not supported on Flutter Web. Consider using either Image.asset or Image.network instead."

Error Shown when I try to upload image to signup page Upload ID Image..

**Pull request merged**